app/templates/components/add-reminder.hbs

```diff
@@ -0,0 +1,16 @@
+{{!-- {{outlet}} --}}
+<form class="add-reminder-form" {{action 'addReminder' on="submit"}}>
+  <label>
+    Title:
+    {{input class='title-input' type='text' value=title}}
+  </label>
+  <label>
+    Date:
+    {{input class='date-input' type='text' value=date}}
+  </label>
+  <label>
+    Notes:
+    {{input class='notes-input' type='text' value=notes}}
+  </label>
+  <button class= 'submit-new-btn' {{action 'addReminder'}}>Submit</button>
```

Comment:
We only need this in one place - I'd suggest keeping it in the form submission and not on click.